ports/stm32/boards/make-pins.py

```diff
@@ -59,6 +59,11 @@ class Stm32Pin(boardgen.Pin):
     def __init__(self, cpu_pin_name):
         super().__init__(cpu_pin_name)
 
+        # Pins ending in "_C" are special analog-only pins.
```

Comment:
Can you add "H7" and "Dual-pad" to this comment -- if you don't know about dual pad, it would be very difficult to figure out what an "analog-only pin" is.

Reply:
Done.